doc/Administration-guide/System-Configuration/General-configuration/index.md

```diff
@@ -4,4 +4,183 @@ title: General configuration
 permalink: /:path/
 sort_idx: 10
 ---
-Kaa node configuration. Inter-node communication and networking config.
+
+After Kaa installation on Ubuntu/Debian OS (deb packages), configuration files for each Kaa component will be extracted into the
```

Comment:
Please check installation guide and add (if this is different) info for CentOS/RHEL